scripts/db-exporter/build.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# The script must be invoked with the concordium-node source directory as working directory.
+
+# Expected parameters:
+# - VERSION
+
+version="${VERSION}"
+
+rust_toolchain_version=1.62
+protoc_version=3.15.3
+
+# Install dependencies.
+
+apt-get update && \
+	DEBIAN_FRONTEND=noninteractive apt-get -y install \
+		curl \
+		libunbound-dev \
+		libprotobuf-dev \
+		libssl-dev \
+		pkg-config \
+		libnuma-dev \
+		libgmp-dev \
+		liblmdb0 \
+		postgresql-server-dev-all \
+		locales \
+        unzip \
+		liblmdb-dev
+
+# Install protobuf
+
+curl -L https://github.com/protocolbuffers/protobuf/releases/download/v${protoc_version}/protoc-${protoc_version}-linux-x86_64.zip -o protoc.zip
+unzip protoc.zip bin/protoc -d /usr/
+rm protoc.zip
+
+# Install Haskell.
+
+curl -sSL https://get.haskellstack.org/ | sh
+
+# Install Rust.
+
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source "$HOME/.cargo/env"
+rustup set profile minimal
+rustup default "$rust_toolchain_version"
+
+# - 'database-exporter'
+
+stack build \
+	--stack-yaml=./concordium-consensus/stack.yaml \
+	--flag=concordium-consensus:-dynamic \
+	--copy-bins \
+	--local-bin-path=. \
+	concordium-consensus:database-exporter
```

Comment:
An alternative to building the exporter here would be to reuse the `build-static-libraries.sh` script which also builds this binary and puts it into `/binaries` in the resulting archive.

Then you also would not have to copy libraries from `concordium-base/rust-src/target` and wasm-chain integration.

Reply:
Have you considered that?

Reply:
Changed the build to use the `build-static-libraries.sh`  script, it makes the build take a bit longer, but I think that's fine as it should reduce the amount of maintenance required to keep various builds in line.